ddpui/api/admin_portal/organizations_api.py

```
@@ -0,0 +1,42 @@
+from ninja import Router
+from django.http import Http404
+from ddpui.models.org import Org
+
+admin_org_router = Router()
+
+
+@admin_org_router.get("/v1/organizations")
+@admin_org_router.get("/v1/organizations/")
+def get_admin_organizations(request, name: str = None):
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)